core/domain/feedback_services.py

```diff
@@ -473,7 +496,7 @@ def get_thread_summaries(user_id, thread_ids):
 
     last_two_messages_ids = []
     for thread in threads:
-        last_two_messages_ids += thread.get_last_two_message_ids()
+        last_two_messages_ids += get_last_two_message_ids(thread)
```

Comment:
Hm. What is the rationale for moving this method out of the Thread class and into a function? It seems like it makes more sense within the domain object.

Reply:
Thanks for explaining! I thought about it, and I would suggest that it's probably better to keep it in the Thread class, because it depends on those two fields and we won't be using it to generate arbitrary or hypothetical message IDs. (I would agree with your reasoning if we were doing so, but if it's always going to be the last two message IDs of some actual Thread object, it makes more sense to have that be part of the behaviour of that domain object.)

Reply:
Done.